docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+## Text
+
+Therae are three Text components to display and edit text on the screen:
+
+- The [TextLabel](#textLabel) displays a short text string.
+
+- The [TextField](#textField) is to edit a single line text.
+
+- The [TextEditor](#textEditor) is to edit multi-line text.
+
+<a name="textLabel"></a>
+## TextLabel
+
+The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents.TextLabel` namespace contains the class.
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](hello-world.md).
+
+**Figure: Text label positioned to top left**
+
+![Text label example, positioned to top left](media/TextLabelTopLeft.png)
+
+<a name="create"></a>
+### Create Text Label
+
+To create a text label:
+
+1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+
+    ```csharp
+    TextLabel label = new TextLabel("Hello World");
+    ```
+
+    You can also create the `Tizen.NUI.BaseComponents.TextLabel` class instance separately and define the label text by setting its `Text` property:
+
+    ```csharp
+    TextLabel label = new TextLabel();
+    label.Text = "Hello World";
+    ```
+
+    > [!CAUTION]
+    >
+    > To display the label properly, the `Text` property must be a UTF-8 string. Any `CR+LF` new line characters are replaced by `LF`.
+
+2.  Define the label position on-screen with the `ParentOrigin` property of the `Tizen.NUI.BaseComponents.TextLabel` class:
+
+    ```csharp
+    label.ParentOrigin = ParentOrigin.TopLeft;
+    ```
+
+3.  Add the text label control to a window:
+
+    ```csharp
+    Window window = Window.Instance;
+    window.Add(label);
+    ```
+
+    > [!CAUTION]
+    >
+    > The `TextLabel` object can be only added to a `Window` object or a `View` object on a `Window` object.

```

Comment:
I don't think that we add object is right in this context, we add controls actually on UI. 
The object and its creation is done while writing the code/logic for the control. 

@JoogabYun  what are your thoughts? 
Please correct me if I am wrong.